### PR TITLE
Add some detail to docstrings for special functions

### LINKED ISSUE #4956: Documentation error in `scipy.special.bi_zeros`
The documentation for this function says

    Compute the zeros of Airy Functions Bi(x) and Bi’(x), b and b’ respectively, and the associated values of Ai(b’) and Ai’(b).

Those last two should evidently be `Bi` and `Bi'`:

    Compute the zeros of Airy Functions Bi(x) and Bi’(x), b and b’ respectively, and the associated values of Bi(b’) and Bi’(b).

I've checked that the latter is the actual behavior of this function with this code:

```python
import scipy.special as sps
n = 20
bi_z = sps.bi_zeros(n)
for k in range(n):
    bi, bip = bi_z[0][k], bi_z[1][k]
    Bi_bi, Bip_bi = sps.airy(bi)[2:]
    Bi_bip, Bip_bip = sps.airy(bip)[2:]
    assert abs(Bi_bi)  < 3e-12, ('Bi_bi',k,Bi_bi)
    assert abs(Bip_bip) < 5e-11, ('Bip_bip',k,Bip_bip)
    assert abs(Bip_bi-bi_z[3][k]) < 3e-12, ('Bip_bi',k,Bip_bi,bi_z[3][k])
    assert abs(Bi_bip-bi_z[2][k]) < 5e-11, ('Bi_bip',k,Bi_bip,bi_z[2][k])
```

### LINKED ISSUE #4957: Docstring for `pbvv_seq` is wrong
[The docstring](https://github.com/scipy/scipy/blob/v0.14.0/scipy/special/basic.py#L956) for `pbvv_seq` seems to have been copied from `pbdv_seq`.  It currently says

    Compute sequence of parabolic cylinder functions Dv(x) and their derivatives for Dv0(x)..Dv(x) with v0=v-int(v).

It should say

    Compute sequence of parabolic cylinder functions Vv(x) and their derivatives for Vv0(x)..Vv(x) with v0=v-int(v).

I checked that the function gives the latter behavior with

```python
import numpy as np
import scipy.special as special
N = 10
for z in 10*np.random.random(20)-5:
    assert np.allclose(np.array(special.pbvv_seq(N, z)), np.array([special.pbvv(n, z) for n in range(N+1)]).T, rtol=2e-12, atol=0)
```

### PR DESCRIPTION
**Pull request by moble**
The `bi_zeros` and `pbvv_seq` docstrings were incorrectly copied from `ai_zeros` and
`pbdv_seq`, respectively.  This corrects those docstrings and adds some more
detail.

The `specfun` functions all come from "Computation of Special Functions" by
Zhang and Jin, so I've added a reference to that work to each function that
uses `specfun`.

This closes #4956 and closes #4957.

### CODE REVIEW

**Comment by ev-br** [comment]
Thanks! Can you also run `cd special; ./generate_unfuncs.py` and commit the result (an appropriate commit message would be "GEN: regenerate _ufuncs.pyx"). Here's an explanation from Pauli of what this is: https://github.com/scipy/scipy/pull/2905#issuecomment-24835960

As a nitpick, these all functions cannot all come from Chapter 13 of the Zhang and Jun book, can they? If you have a book, it could be a bonus to add actual chapter numbers, but otherwise it's already an improvement.

**Comment by moble** [comment]
Sorry, I thought I removed those chapters when pasting, but I guess I forgot.  That's not a nitpick at all.  :)    I've corrected some, and removed others.  Also added some `Parameters` sections to a few functions I wasn't clear on, and ran `generate_unfuncs.py`.

File: scipy/special/basic.py [hunk 1]
```diff
@@ -1065,6 +1628,13 @@ def kelvin_zeros(nt):
     length 8 tuple of arrays of length nt.
     The tuple containse the arrays of zeros of
```
Comment (ev-br) [inline scipy/special/basic.py]: typo: containSE

**Comment by moble** [comment]
I've gone through, and shortened all the first lines.  Some of those "first line"s were huge; I preserved the information by just moving it lower in the docstring, while still trying to make the summary sufficiently descriptive.  I haven't checked literally everything, but it compiles and everything I've looked at renders properly.

**Comment by ev-br** [comment]
LGTM. I intend to merge it in a couple of days unless there are further comments. 

In an ideal world, there's a verified reference to A&S and/or DLMF for each special function, but that's clearly out of scope for this PR :-).

**Comment by moble** [comment]
> In an ideal world, there's a verified reference to A&S and/or DLMF for each special function,

It'd be nice to have, since they're online and free.  But it's important to note that these `specfun` functions really come from Zhang and Jin, and they don't use the same notation as DLMF.  For example, the parabolic cylinder functions use conventions that I can't find elsewhere -- even DLMF's [listing of alternative notations](http://dlmf.nist.gov/12.1) isn't consistent with the usage of Zhang and Jin (and hence `scipy.special`).  So if anyone does come along and add those references later, it'd be good to point out that the notation may not be the same, or even figure out when the notation is and is not the same.

> but that's clearly out of scope for this PR :-).

Agreed.  :)

**Comment by ev-br** [comment]
I do not disagree with you, adding the reference to Zhang&Jin is a clear improvement. All I'm saying is that the facts that the notation is different and that the book is not very easy to find IMO makes going over the usage and adding conversion formulas only more valuable.

**Comment by ev-br** [comment]
OK, no objections for six days, I'll take it as a silent yes. Merging. Thank you @moble, keep them coming!

**Pull request merged**